spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowJobExecutionDao.java

```diff
@@ -42,13 +42,9 @@
 	 * SQL statements for removing the Step Execution Context.
 	 */
 	private static final String  SQL_DELETE_BATCH_STEP_EXECUTION_CONTEXT =
-			"DELETE FROM %PREFIX%STEP_EXECUTION_CONTEXT " +
-			"WHERE STEP_EXECUTION_ID " +
-			"IN ( " +
-			"SELECT SEC.STEP_EXECUTION_ID " +
-			"FROM %PREFIX%STEP_EXECUTION_CONTEXT SEC " +
-			"JOIN %PREFIX%STEP_EXECUTION SE ON SE.STEP_EXECUTION_ID = SEC.STEP_EXECUTION_ID " +
-			"WHERE SE.JOB_EXECUTION_ID in (:jobExecutionIds))";
+			"DELETE SEC.* FROM BATCH_STEP_EXECUTION_CONTEXT SEC " +
```

Comment:
@ghillert Could you please some description in the commit to better understand what went wrong and what is fixed with this PR? Thanks!